observer/test/observer.test.js

```diff
@@ -0,0 +1,64 @@
+import assert from 'node:assert'
+import { observeScheduledRewards } from '../lib/observer.js'
+import { getPgPools } from '@filecoin-station/spark-stats-db'
+
+describe('observer', () => {
+  describe('observeScheduledRewards', () => {
+    let pgPools
+
+    before(async () => {
+      pgPools = await getPgPools()
+    })
+
+    it('observes scheduled rewards', async () => {
+      await pgPools.evaluate.query('DELETE FROM daily_participants')
+      await pgPools.evaluate.query('DELETE FROM participants')
+      const { rows: insertRows } = await pgPools.evaluate.query(`
+        INSERT INTO participants
+        (participant_address)
+        VALUES
+          ('0xCURRENT'),
+          ('0xOLD')
+        RETURNING id
+      `)
```

Comment:
Let's extract this query to a helper function, please. See the existing similar helpers for inspiration:

https://github.com/filecoin-station/spark-stats/blob/831e6694f6515b338afb0ace1c65acdd8cc492c3/stats/test/platform-routes.test.js#L197-L220

https://github.com/filecoin-station/spark-stats/blob/831e6694f6515b338afb0ace1c65acdd8cc492c3/stats/test/handler.test.js#L353-L370

Reply:
In fact, I think you can replace both of these SQL queries with a single call to `givenDailyParticipants()`.

Reply:
sgmt, but how would you propose to share this function between the packages?

Reply:
move to spark-evaluate